client/components/navigation/Navbar.tsx

```diff
@@ -39,7 +60,72 @@ function Navbar() {
           </HStack>
         </NextLink>
 
-        <HStack spacing={{ base: 2, sm: 7 }}>
+        <IconButton
+          aria-label=""
```

Comment:
We should conditionally add a label here!

```suggestion
          aria-label={hamburger.isOpen ? 'Close menu' : 'Open menu'}
```